events/models.py

```diff
@@ -467,7 +467,7 @@ class Enrolment(TimestampedModel):
         on_delete=models.CASCADE,
         verbose_name=_("occurrence"),
     )
-    attended = models.NullBooleanField(verbose_name=_("attended"))
+    attended = models.BooleanField(verbose_name=_("attended"), null=True, blank=True)
```

Comment:
Shouldn't / couldn't this be default=False instead of blank=True and null=True?

Reply:
I think this is something binary: you have attended or you have not. When answering null, the user or the system says that he does not want to answer or has not yet answered the question, but what is its benefit at this point?